leanpkg.toml

```diff
@@ -6,3 +6,4 @@ path = "src"
 
 [dependencies]
 mathlib = {git = "https://github.com/leanprover/mathlib", rev = "b37526407ed1846f29dcdfb976554daf7f8a983e"}
+mason-stother = {git = "https://github.com/johoelzl/mason-stother", rev = "a4095993c2f7fc58091f1a1d05459eae17f786b1"}
```

Comment:
This import is a nightmare (at least for me) -- it's full of errors when I try to compile it. Are we sure we need it? Can we work around it somehow?

Reply:
OK! How about we even sorry it but leave the actual definition commented out, with some comment about how it will all work once polynomials in 1 variable are in mathlib.

Reply:
Ok, I
* commented out the mason-stother stuff,
* sorried the definition of `is_integral`, and
* removed the dependency on mason-stother from the `toml`.